redaxo/src/core/lib/util/i18n.php

```diff
@@ -277,9 +277,12 @@ public static function translate($text, $use_htmlspecialchars = true, callable $
         $transKeyLen = strlen($tranKey);
         if (substr($text, 0, $transKeyLen) == $tranKey) {
             if (!$i18nFunction) {
-                $i18nFunction = $use_htmlspecialchars ? 'self::msg' : 'self::rawMsg';
+                if ($use_htmlspecialchars) {
+                    return self::msg(substr($text, $transKeyLen));
+                }
+                return self::rawMsg(substr($text, $transKeyLen));
             }
-            return call_user_func($i18nFunction, substr($text, $transKeyLen));
+            return $i18nFunction(substr($text, $transKeyLen));
```

Comment:
Sorry, für's späte Kommentieren, aber ich glaube nicht alle gültigen `callables` lassen sich per `$func()` aufrufen, zumindest nicht in PHP 5. Zum Beispiel Strings der Form `foo::bar`, glaube ich.

Reply:
Stimmt https://3v4l.org/D4JZ4

Reply:
gerade nochmal getestet: https://3v4l.org/fjmFU

unter php5 ist ' foo::bar' auch kein callable, daher gibts hier in meinen Augen kein Problem unter php5 (wg. dem typehint am parameter)

Reply:
hmm, seit 5.4 ist es doch ein callable? Der Fehler kommt doch nur in <=5.3..

Reply:
followup PR mit regression test: https://github.com/redaxo/redaxo/pull/1983